Makefile

```diff
@@ -172,40 +172,57 @@ OpenRA.Editor.Form1.resources:
 editor: OpenRA.Editor.MapSelect.resources OpenRA.Editor.Form1.resources $(editor_TARGET)
 
 check:
```

Comment:
Make this (and the `test` target below) depend on the `utility` target (see how the `docs` target does it if you don't know how that works).